apps/web/lib/integrations/calendar/services/VimcalCalendarService.ts

```diff
@@ -0,0 +1,329 @@
+import { Credential, Prisma } from "@prisma/client";
```

Comment:
Let's find a way to implement this without duplicating all of the Google Calendar code